src/k-provider/ovp/loaders/entry-list-loader.js

```diff
@@ -53,7 +53,15 @@ export default class OVPEntryListLoader implements ILoader {
     const config = OVPConfiguration.get();
     const requests: Array<RequestBuilder> = [];
     params.entries.forEach(entry => {
-      requests.push(OVPBaseEntryService.list(config.serviceUrl, params.ks, entry.entryId || entry, params.redirectFromEntryId));
+      requests.push(
+        OVPBaseEntryService.list(
+          config.serviceUrl,
+          params.ks,
+          entry.entryId || (typeof entry === 'string' && entry),
```

Comment:
this means you pass `false` when `typeof entry !== 'string'`. while you have to pass `string`

Reply:
No, either entry is object with entryId prop or entry is a string

Reply:
so you can remove the `typeof` check

Reply:
Yep, it also fixed flow